vms/avm/vm.go

```diff
@@ -720,6 +729,13 @@ func (vm *VM) verifyFxUsage(fxID int, assetID ids.ID) bool {
 	// make a map
 	for _, state := range createAssetTx.States {
 		if state.FxID == uint32(fxID) {
+			// Cache that this asset supports this fx
+			fxIDs := ids.BitSet(0)
+			if assetInCache {
+				fxIDs = fxIDsIntf.(ids.BitSet)
+			}
+			fxIDs.Add(uint(fxID))
+			vm.assetToFxCache.Put(assetID, fxIDs)
```

Comment:
Lets make this so that we cache negatives as well. It's more of a code cleanup thing than a performance thing. (essentially implementing the TODO outside of this loop)

Reply:
added this